<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="treinamentoPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;xi bi si&lt;/p&gt;&lt;p&gt;α θ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="testeLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="treinamentoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>67</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Treinamento</string>
    </property>
   </widget>
   <widget class="QLabel" name="testeLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Teste</string>
    </property>
   </widget>
   <widget class="QLabel" name="saidaLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Saida</string>
    </property>
   </widget>
   <widget class="QLabel" name="respostaLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="testeLabel_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="treinamentoPushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>vai</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testePushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>vai2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hebbRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hebb</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="perceptronRadioButton">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>10</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Perceptron</string>
    </property>
   </widget>
   <widget class="QLabel" name="peso1Label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>W1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="peso2Label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>228</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>W2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pesoBLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>247</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>W3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="p1Label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="p2Label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>228</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pBLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>247</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
